Packs/HealthCheck/ReleaseNotes/3_0_0.md

```diff
@@ -0,0 +1,132 @@
+
```

Comment:
I made the RN format, please replace each - %%UPDATE_RN%% with the related release note.